### PR TITLE
Add Case Type( Feature2 )

### PR DESCRIPTION
**Pull request by selva08sankari**
- case type added
- case name is validated
- cost of case is validated

### CODE REVIEW

File: src/main/java/in/casekartin/service/CaseManagerService.java [hunk 1]
```diff
@@ -48,4 +49,23 @@ private CaseManagerService() {
...
+	 * 
+	 * @param caseName
+	 * @param cost
+	 * @return
+	 */
+	public static boolean addCaseType(String caseName, String cost) {
+		Float price = Float.parseFloat(cost);
+		if (CaseManagerValidator.caseNameValidator(caseName) && CaseManagerValidator.costValidator(price)
+				&& CaseManagerValidator.isNotExist(caseName)) {
+			CaseManager case0 = new CaseManager(caseName.toUpperCase(), price);
```
Comment (MohamedHalith001) [inline src/main/java/in/casekartin/service/CaseManagerService.java:63]: Rename case0 to mobileCase or phoneCase

File: src/main/java/in/casekartin/validator/CaseManagerValidator.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+					|| (caseName.charAt(i) >= 'a' && caseName.charAt(i) <= 'z') || caseName.charAt(i) == ' ') {
+				isValid = true;
+			}
+			i++;
+		}
+		return isValid;
+	}
+
+	/**
+	 * check whether cost is greater than 100
```
Comment (MohamedHalith001) [inline src/main/java/in/casekartin/validator/CaseManagerValidator.java:43]: Check method description says "check whether cost is greater than 100" but checks for 0

File: src/main/java/in/casekartin/service/CaseManagerService.java (hunk 1 above)
Comment (selva08sankari) [reply src/main/java/in/casekartin/service/CaseManagerService.java:63]: ok. I will change it

File: src/test/java/in/casekartin/CaseManagerValidatorTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	 */
+	@Test
+	public void caseNameWithCorrectInput() {
+		String caseName = "led case";
+		boolean isNameValid = CaseManagerValidator.caseNameValidator(caseName);
+		assertEquals(true, isNameValid);
+	}
+
+	/**
+	 * test case for checking case cost is greater than 100
```
Comment (MohamedHalith001) [inline src/test/java/in/casekartin/CaseManagerValidatorTest.java:46]: Change

**Pull request merged**